affiliatewp-order-details-for-affiliates.php

```diff
@@ -303,7 +339,7 @@ public function get_theme_template_paths( $file_paths ) {
 	}
 
 	/**
-	 * Determins if the given user can access the purchase details?
+	 * Determins if the given user can access the purchase details.
```

Comment:
```suggestion
	 * Determines if the given user can access the purchase details.
```
Typo in the DocBlock summary.